themes/navy/source/css/_partial/index.styl

```diff
@@ -45,6 +45,39 @@
   @media mq-normal
     display: block
 
+#intro-news-list
+  background-color: #10171e
```

Comment:
"black" is sufficient.

Reply:
I have tried black but it seems not fit navy theme design language.

Reply:
My poor monitor can't differentiate black and #10171e

Anyway, why not use `lighten(color-navy, 10%)` as used in #banner-start-command.

Reply:
The `#10171e` I choose is more like `darken(color-navy, 10%)`. Maybe `lighten(color-navy, 10%)` is better.

Reply:
I'm a bit reluctant to introduce new color. If darker background is preferred, then it's better to use `darken(color-navy, 10%)`.

Reply:
Updated.